rclcpp/include/rclcpp/executors/executor_notify_waitable.hpp

```diff
@@ -140,6 +140,8 @@ class ExecutorNotifyWaitable : public rclcpp::Waitable
   get_number_of_ready_guard_conditions() override;
 
 private:
+  std::atomic<bool> was_triggered = false;
```

Comment:
nitpick: this has to have a trailing underscore

```suggestion
  std::atomic<bool> was_triggered_ = false;
```

And updated elsewhere.

Reply:
@wjwwood Fixed and rebased to master.